media-plugins/gst-plugins-hls/gst-plugins-hls-1.22.1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+GST_ORG_MODULE=gst-plugins-bad
+
+inherit gstreamer-meson
+
+DESCRIPTION="HTTP live streaming plugin for GStreamer"
```

Comment:
There is a new adaptivedemux2 element in 1.22, which replaces HLS, DASH and MSS adaptive demuxing cases. We might want to look into packaging it.